includes/checkout.php

```diff
@@ -205,11 +226,6 @@ function pmpro_complete_async_checkout( $order ) {
 		'enddate'         => $enddate
 	);
 
-	global $pmpro_error;
```

Comment:
This seems like a good thing to remove. I can imagine it might fail for onsite checkouts where ipnlog is not defined. At the same time, was this at all useful for debugging ipn/webhook logs?

Reply:
The `ipnlog()` function was only ever used to log error messages for PayPal Standard checkout completions. I'm not concerned about removing it, and we can add it back to the PayPal Standard code specifically if needed.